PixelHunter1995/FontManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace PixelHunter1995
+{
+    class FontManager
+    {
+        private Dictionary<string, SpriteFont> Fonts;
+        private static FontManager instance;
+
+        public static FontManager Instance
+        {
+            get
+            {
+                if (instance == null)
+                {
+                    instance = new FontManager();
+                }
+                return instance;
+            }
+        }
+
+        private FontManager() {}
+
+        public SpriteFont getFontByName(string fontName)
+        {
+            return Fonts[fontName];
+        }
+
+        public void LoadContent(ContentManager content)
+        {
+            Fonts = new Dictionary<string, SpriteFont>();
+            // Add new fonts here
```

Comment:
Do like in SceneManager? "foreach (string filepath in Directory.GetFiles(scenesPath))"

Reply:
Can't, since we don't copy the font files to the output dir. We just rely on them being built in the Content.mgcb file. We could copy them, but that would be redundant and confusing...